appveyor.yml

```diff
@@ -1,21 +1,23 @@
 os: Visual Studio 2015
 
-platform:
-  - x64
-  - x86
-
 environment:
-  NODEJS_VERSION: "6"
   RUST_BACKTRACE: 1
   matrix:
-    - NODE_ARCHITECTURE: x64
+    - PLATFORM: x64
+      NODEJS_VERSION: "6"
       RUST_TOOLCHAIN: stable-x86_64-pc-windows-msvc
-
-    - NODE_ARCHITECTURE: x86
+    - PLATFORM: x86
+      NODEJS_VERSION: "6"
+      RUST_TOOLCHAIN: stable-i686-pc-windows-msvc
+    - PLATFORM: x64
+      NODEJS_VERSION: "8"
+      RUST_TOOLCHAIN: stable-x86_64-pc-windows-msvc
+    - PLATFORM: x86
+      NODEJS_VERSION: "8"
```

Comment:
Thanks, yeah I wanted to test both LTS + latest and the old configuration was doing a 2x2 matrix where you'd test x64 toolchain on a x86 machine.